scripts/build/build_node/Platform/Linux/package-list.ubuntu-focal.txt

```diff
@@ -21,3 +20,4 @@ libxcb-xfixes0-dev                      # For mouse input
 libxcb-xinput-dev                       # For mouse input
 zlib1g-dev
 mesa-common-dev
+ros-dev-tools                           # ROS2 development tools
```

Comment:
```suggestion
ros-humble-ackermann-msgs 
ros-humble-control-toolbox 
ros-humble-gazebo-msgs 
ros-humble-tf2-ros
ros-humble-urdfdom
ros-humble-vision-msgs
````

Reply:
That's fair, are these the minimum sets or would a developer need more from `ros-dev-tools` to debug?